setup/setup.sh

```diff
@@ -1,25 +1,47 @@
 #!/usr/bin/env bash
 
-. ./env.sh
+if [ -z "$1" ]
+  then
+    echo "Please provide the OpenShift login Id."
+    echo "  e.g. ./setup/setup.sh developer"
+    echo ""
+    exit 1
+fi
+
+. ./env.sh $1 
 
 oc login -u $USER
 
-#oc delete project $CICD_PROJECT
-oc new-project $CICD_PROJECT 2> /dev/null
+
+oc delete project $DEV_PROJECT
+oc new-project $DEV_PROJECT 2> /dev/null
 while [ $? \> 0 ]; do
     sleep 1
     printf "."
-    oc new-project $CICD_PROJECT 2> /dev/null
+    oc new-project $DEV_PROJECT 2> /dev/null
 done
 
 oc new-app jenkins-persistent
 
-#oc delete project $DEV_PROJECT
-oc new-project $DEV_PROJECT 2> /dev/null
+echo "Setup the surrounding softwate and environment"
+echo
+echo "Start up MySQL for database access"
+
+oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mysql-ephemeral-template.json
+oc new-app --template=mysql-ephemeral --param=MYSQL_PASSWORD=password --param=MYSQL_USER=dbuser --param=MYSQL_DATABASE=sampledb
```

Comment:
You should probably not create the template locally. Instead, you should try:

`oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mysql-ephemeral-template.json -p MYSQL_PASSWORD=password -p MYSQL_USER=dbuser -p MYSQL_DATABASE=sampledb`